samples/matter/thermostat/src/temperature_measurement/real_temp.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+#include <stdint.h>
+#include "binding_handler.h"
+#include <platform/CHIPDeviceLayer.h>
+
+
+class RealSensor 
+{
+  public:
```

Comment:
Please install "clang-format" and integrate it in VS Code (or whatever editor you use) so that you automatically apply the style defined in `samples/matter/.clang-format` and save your time on code formatting :)